tests/foreman/cli/test_contentviews.py

```diff
@@ -1832,19 +1749,14 @@ def test_cv_roles_admin_user_negative(self, test_data):
         @status: Manual
 
         """
-
         password = gen_alphanumeric()
         no_rights_user = make_user({'password': password})
         no_rights_user['password'] = password
 
-        org_obj = make_org(cached=True)
-
-        result = Org.info({'id': org_obj['id']})
-        self.assertEqual(result.return_code, 0, "Failed to create object")
-        self.assertEqual(
-            len(result.stderr), 0, "There should not be an exception here")
-
-        test_data['organization-id'] = org_obj['id']
+        try:
+            test_data['organization-id'] = make_org(cached=True)['id']
```

Comment:
Do we only want the ``id`` here? Don't we need a dictionary for the next time this is used?